models/error.go

```diff
@@ -887,3 +887,28 @@ func IsErrExternalLoginUserNotExist(err error) bool {
 func (err ErrExternalLoginUserNotExist) Error() string {
 	return fmt.Sprintf("external login user link does not exists [userID: %d, loginSourceID: %d]", err.UserID, err.LoginSourceID)
 }
+
+//  ________      _____   ____ _________________ ___
+//  \_____  \    /  _  \ |    |   \__    ___/   |   \
+//   /   |   \  /  /_\  \|    |   / |    | /    ~    \
+//  /    |    \/    |    \    |  /  |    | \    Y    /
+//  \_______  /\____|__  /______/   |____|  \___|_  /
+//          \/         \/                         \/
+
```

Comment:
Maybe you could take the chance to start with splitting error.go into error_oauth.go ...